src/rest/api.go

```diff
@@ -3,15 +3,16 @@ package rest
 import (
 	"config"
 	"encoding/json"
-	"github.com/DeKugelschieber/go-resp"
-	"github.com/DeKugelschieber/go-session"
-	log "github.com/sirupsen/logrus"
 	"instance"
 	"model"
 	"net/http"
 	"settings"
 	"strconv"
 	"user"
+
+	resp "github.com/DeKugelschieber/go-resp"
+	session "github.com/DeKugelschieber/go-session"
```

Comment:
`resp` and `session` is not required, since those are the names of the packages already.